jib-maven-plugin/pom.xml

```diff
@@ -293,6 +293,60 @@
               </resources>
             </configuration>
           </execution>
+
+          <!-- Imports jib-plugins-common sources -->
```

Comment:
I think we could combine this with the jib-core source adding so that it just imports both sources (and the comment then becomes like `Imports jib-core and jib-plugins-common sources`